runner.go

```
@@ -203,12 +209,6 @@ func Run(options ...RunOption) {
 	executorOptions := append([]ExecutorOption{WithWatchMode(watchMode)}, runtime.executorOptions...)
 	executor := NewExecutor(c, tasks, executorOptions...)
 
-	if len(tasks) == 0 {
-		log.Fatalln("No tasks specified")
-	}
-	log.Println("Desired tasks:", strings.Join(desiredTasks, ", "))
-	log.Printf("Watch mode: %t", watchMode)
-
```

Moved up to fail fast.